cmd/aries-agent-mobile/pkg/api/notifier.go

```diff
@@ -0,0 +1,17 @@
+/*
+ Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+// Notifier represents a notification dispatcher.
+type Notifier interface {
+
+	// Notify sends the given message to all of the subscribers.
+	Notify(topic string, message []byte) error
```

Comment:
remove this method `Notify `.
Only `Execute(topic string, message []byte)` method should be here.
You can rename `Execute` to `Notify ` if you want.

Reply:
ok, will do. thought so too